django/views/debug.py

```diff
@@ -350,6 +350,12 @@ def get_traceback_data(self):
             c['request_COOKIES_items'] = self.request.COOKIES.items()
             c['request_insecure_uri'] = self._get_raw_insecure_uri()
 
+            try:
+                view_name = self.request.resolver_match._func_path
+            except Exception:
+                view_name = None
+            c['raising_view_name'] = view_name
```

Comment:
IMO we should add a new hook, e.g. `get_caller(resolver_match)` with the logic from `technical_404_response`:
https://github.com/django/django/blob/c67e1cf44f17c36139e25b1eae92216cb8baad77/django/views/debug.py#L539-L556

and use it in both views. For example:
```python
def get_caller(resolver_match):
    obj = resolver_match.func
    caller = ''
   
    if hasattr(obj, 'view_class'):
        caller = obj.view_class
    elif hasattr(obj, '__name__'):
        caller = obj.__name__
    elif hasattr(obj, '__class__') and hasattr(obj.__class__, '__name__'):
        caller = obj.__class__.__name__
        
    if hasattr(obj, '__module__'):
        module = obj.__module__
        caller = '%s.%s' % (module, caller)
    return caller


```

Reply:
I agree. I've added this hook keeping #15292 in mind. I'll be adding tests soon.